src/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
When you change class(modification) in HTML, the count of active stars should change.